### PR TITLE
created alumni form

### PR DESCRIPTION
**Pull request by Sharma-palak**
@bvishal8510 sir,@SukhnandanMalhotra  maam,@deepanshu-jain1999 sir plzz review the pull request.

### CODE REVIEW

File: Core/admin.py [hunk 1]
```diff
@@ -40,3 +40,4 @@ class YearAdmin(admin.ModelAdmin):
 admin.site.register(Blog)
 admin.site.register(EmailAttachment)
 admin.site.register(EmailContent)
+admin.site.register(AlumniRegistration)
```
Comment (bvishal8510) [inline Core/admin.py:43]: Add a newline at end of file

File: Core/models.py [hunk 2]
```diff
@@ -244,10 +244,27 @@ def __str__(self):
...
                                 default=None)
     name = models.CharField(max_length=80, blank=False, validators=[validate_file_extension])
 
     def __str__(self):
         return self.event.name + "  " + self.name
+
+
+class AlumniRegistration(models.Model):
+    choice = (
+         ('22 September(Sunday)','21 September(Sunday)'),
```
Comment (bvishal8510) [inline Core/models.py]: Why are both dates different? Cannot have Sunday on both 22 and 21.

File: Core/models.py [hunk 3]
```diff
@@ -244,10 +244,27 @@ def __str__(self):
...
+         ('29 September(Sunday)','28 September(Sunday)'),
```
Comment (bvishal8510) [inline Core/models.py]: Same here

File: Core/models.py [hunk 4]
```diff
@@ -244,10 +244,27 @@ def __str__(self):
...
+     )
+    name = models.CharField(max_length=200)
+    batch = models.CharField(max_length=4)
+    phone_regex = RegexValidator(regex=r"^[56789]\d{9}$")
+    contact_no = models.CharField(validators=[phone_regex], max_length=10, blank=True, null=True)
+    message = models.CharField(max_length=500,blank=True,null=True)
+    date = models.CharField(max_length=200,choices=choice,blank=True,null=True)
+
+    def __str__(self):
+        return self.name
```
Comment (bvishal8510) [inline Core/models.py:270]: Add an empty line at the end of file

File: Core/validators.py [hunk 5]
```diff
@@ -6,4 +6,3 @@ def validate_file_extension(value):
     ext = os.path.splitext(value)[1]
     if ext.lower() != '.pdf':
         raise ValidationError(u'File extension should be pdf!')
-
```
Comment (bvishal8510) [inline Core/validators.py:9]: Add newline at end of file

File: Core/views.py [hunk 6]
```diff
@@ -205,6 +206,35 @@ def get(self, request, *args,**kwargs):
         return render(request, self.template_name)
 
 
+class AlumniRegistrationView(View):
+    template_name = "alumni-registration.html"
+
+    def post(self,request,*args,**kwargs):
+        form = RegistrationAlumni(request.POST)
+        if form.is_valid():
+            # alumni = form.save(commit=False)
```
Comment (bvishal8510) [inline Core/views.py]: remove all comments

File: Core/views.py [hunk 7]
```diff
@@ -205,6 +206,35 @@ def get(self, request, *args,**kwargs):
...
+            # print(alumni.id)
+            # alumni_obj = AlumniRegistration.objects.get(pk=alumni.id)
+            # print(alumni_obj)
+            # alumni_obj.confirmation = bool(int(confirmation))
+            # print(alumni_obj.confirmation)
+            # alumni_obj.save()
+            form.save()
+            return redirect('home')
+        else:
+            print('error')
```
Comment (bvishal8510) [inline Core/views.py]: remove print statement

File: IceCream/urls.py [hunk 8]
```diff
@@ -18,7 +18,7 @@
 from django.conf import settings
 from django.contrib import admin
 from Core import views
-from Core.views import view404, view500
+# from Core.views import view404, view500
```
Comment (bvishal8510) [inline IceCream/urls.py]: Why are handlers removed?

**Comment by bvishal8510** [comment]
Make sure there is only one migration file for this commit.

**Comment by Sharma-palak** [comment]
@bvishal8510 sir,@SukhnandanMalhotra maam,@deepanshu-jain1999 sir plzz review the pull request.

**Comment by Sharma-palak** [comment]
@bvishal8510 sir,@SukhnandanMalhotra maam,@deepanshu-jain1999 sir plzz review the changes

File: Core/models.py [hunk 9]
```diff
@@ -244,10 +244,31 @@ def __str__(self):
...
 
     def __str__(self):
         return self.event.name + "  " + self.name
+
+
+class AlumniRegistration(models.Model):
+    choice = (
+         ('22 September(Sunday)','22 September(Sunday)'),
+         ('29 September(Sunday)','29 September(Sunday)'),
+         ('2  October(Sunday)','2 October(Sunday)'),
```
Comment (animesh21) [inline Core/models.py:260]: 2nd of October isn't a Sunday this year.

**Pull request merged**